src/main/java/net/eno/pieces/Color.java

```diff
@@ -0,0 +1,5 @@
+package net.eno.pieces;
+
+public enum Color {
+    WHITE, BLACK
```

Comment:
한 줄에 하나씩 선언해주시는 것이 가독성 측면과 코드스타일 일관성 측면에서 조금 낫습니다.

```suggestion
    WHITE,
    BLACK

Reply:
그렇게 하도록 하겠습니다!! 감사합니다.